src/lib/datepicker/calendar.ts

```diff
@@ -44,7 +44,20 @@ import {MatMonthView} from './month-view';
 import {MatMultiYearView, yearsPerPage, yearsPerRow} from './multi-year-view';
 import {MatYearView} from './year-view';
 import {Directionality} from '@angular/cdk/bidi';
+import {ComponentPortal, ComponentType, Portal} from '@angular/cdk/portal';
 
+/**
+ * Default header of a [MatCalendar].
+ */
+@Component({
+    selector: 'default-header',
```

Comment:
`mat-calendar-header`

Reply:
done in 3467c0ac390cb4401cf7c44b79c1280ba5dcb5f8